README.markdown

```diff
@@ -347,3 +347,15 @@ If setting the `default` alias does not establish the node version in new shells
 [3]: https://travis-ci.org/creationix/nvm
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+## Uninstall
+
+```
+rm -rf $NVM_DIR ~/.npm ~/.bower && \
```

Comment:
Also, it's a very bad idea to remove `~/.npm` unless you never again intend to use node on your machine.